packages/compass-explain-plan/src/components/explain-plan-modal.spec.tsx

```diff
@@ -51,4 +60,48 @@ describe('ExplainPlanModal', function () {
     render({ status: 'ready' });
     expect(screen.getByText('Query Performance Summary')).to.exist;
   });
+
+  it('should show "Interpret for me" button when AI assistant is enabled', function () {
+    render(
+      {
+        status: 'ready',
+        explainPlan: {
+          namespace: 'test',
+          usedIndexes: [],
+        } as any,
+      },
+      { preferences: { enableAIAssistant: true } }
+    );
+    expect(screen.getByTestId('interpret-for-me-button')).to.exist;
+  });
+
+  it('should not show "Interpret for me" button when AI assistant is disabled', function () {
+    render(
+      {
+        status: 'ready',
+        explainPlan: {
+          namespace: 'test',
+          usedIndexes: [],
+        } as any,
+      },
+      { preferences: { enableAIAssistant: false } }
+    );
+    expect(screen.queryByTestId('interpret-for-me-button')).to.not.exist;
+  });
+
+  it('should disable the "Interpret for me" button when the status is not ready', function () {
+    render(
+      {
+        status: 'loading',
+        explainPlan: {
+          usedIndexes: [],
+        } as any,
+      },
+      { preferences: { enableAIAssistant: true } }
+    );
+    expect(screen.getByTestId('interpret-for-me-button')).to.have.attr(
```

Comment:
super nit, but can probably just tack the inverse of this assertion on to the 'should show "Interpret for me" button when AI assistant is enabled' test to make sure it is usually enabled because nothing's checking it currently.